torchtune/models/flux/_model_builders.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+from torchtune.models.flux._autoencoder import FluxAutoencoder
+
+
+def flux_autoencoder() -> FluxAutoencoder:
```

Comment:
Should this be a model builder or component builder? 

Also, should it be named `flux_v1_autoencoder`, anticipating future flux versions? Or just `flux_autoencoder`, anticipating that the autoencoder part probably won't change in the next version?